support-frontend/assets/helpers/forms/paymentMethods.ts

```diff
@@ -44,4 +45,28 @@ export const isPaymentMethod = (
 	);
 };
 
+export function toPaymentMethodSwitchNaming(
```

Comment:
Presume we will eventually retire the same call in `support-frontend/assets/helpers/forms/checkouts.ts`?